README.md

```diff
@@ -1,4 +1,14 @@
-GraphViz [![Build Status](https://secure.travis-ci.org/phpDocumentor/GraphViz.png)](http://travis-ci.org/phpDocumentor/GraphViz)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
```

Comment:
Just wondering: why yellow and not green (since green indicates success and yellow warning)

Reply:
That's just how the badge was written in the list -- https://gist.github.com/lukas-h/2a5d00690736b4c3a7ba